disnake/ui/action_row.py

```diff
@@ -50,25 +55,23 @@ class ActionRow:
     .. versionadded:: 2.4
 
     Parameters
-    ------------
-    *items: :class:`Item`
+    ----------
+    *items: :class:`WrappedComponent`
         The items of this action row.
     """
 
-    def __init__(self, *items: Item):
+    def __init__(self, *items: WrappedComponent):
         self.width: int = 0
         components = []
```

Comment:
This isn't really needed, we'll have to add `# type: ignore` after `components.append(item._underlying)`
This list goes to `ActionRowComponent._raw_construct` anyways

Reply:
`_underlying` should be a `NestedComponent` as well, so that `# type: ignore` could be removed.
And yeah, all the typechecking doesn't really matter when calling `_raw_construct` ~~(pyright/mypy plugin when)~~, but it's still useful to have proper typechecking where possible